src/log-filestore.c

```diff
@@ -228,10 +249,16 @@ static void LogFilestoreLogCreateMetaFile(const Packet *p, const File *ff, char
     FILE *fp = fopen(metafilename, "w+");
     if (fp != NULL) {
         char timebuf[64];
+        char sidbuf[10*PACKET_ALERT_MAX];
```

Comment:
think I'd prefer this to be initialized, then remove the initialization from the CreateSIDString func